deepmd/descriptor/se_a.py

```diff
@@ -164,10 +164,9 @@ def __init__ (self,
         self.embedding_net_variables = None
         self.mixed_prec = None
         self.place_holders = {}
-        nei_type = np.array([])
+        self.nei_type = np.array([])
         for ii in range(self.ntypes):
-            nei_type = np.append(nei_type, ii * np.ones(self.sel_a[ii])) # like a mask 
-        self.nei_type = tf.constant(nei_type, dtype = tf.int32)
+            self.nei_type = np.append(self.nei_type, ii * np.ones(self.sel_a[ii])) # like a mask
```

Comment:
It looks like it should be an int32. However, the default type for `np.ones` is float32.

Reply:
btw, it may be written as
```py
np.repeat(np.arange(self.ntypes), self.sel_a)
```